dozer/cogs/levels.py

```diff
@@ -59,7 +62,7 @@ def level_for_total_xp(xp):
         # > The formula to calculate how many xp you need for the next level is 5 * (lvl ^ 2) + 50 * lvl + 100 with
         # > your current level as lvl
         lvl = 0
-        while xp >= 0:
+        while xp >= 0 and lvl <= 1000000:  # The same limitation is applied here
```

Comment:
If `lvl` is indeed over 1000000 here, this function would end up returning -1, which seems like unintended behavior.